users-service/features/support/env.ls

```diff
@@ -8,7 +8,7 @@ require! {
 db = null
 get-db = (done) ->
   return done db if db
-  MongoClient.connect "mongodb://localhost:27017/space-tweet-users-test", N (mongo-db) ->
+  MongoClient.connect "mongodb://localhost:27017/space-tweet-db-test", N (mongo-db) ->
```

Comment:
Each service gets their own database. They can create several collections if they want to. The whole database gets backed up, migrated, etc.